scripts-available/CDB_Groups_API.sql

```diff
@@ -0,0 +1,57 @@
+-- Requires configuration parameter. Example: SELECT cartodb.CDB_Conf_SetConf('groups_api', '{ "host": "127.0.0.1", "port": 3000, "timeout": 10, "username": "superadmin", "password": "monkey" }');
+
+-- Sends the create group request
+CREATE OR REPLACE
+FUNCTION cartodb._CDB_Group_CreateGroup_API(database_name text, group_name text, group_role text)
+    RETURNS VOID AS
+$$
+    import httplib
+    import base64
+    import string
+    import json
+
+    try:
+      conf = plpy.execute("SELECT cartodb.CDB_Conf_GetConf('groups_api') conf")[0]['conf']
```

Comment:
Another option we have is using hstore: http://www.postgresql.org/docs/9.3/static/hstore.html

Or something like:

```
CREATE TABLE IF NOT EXISTS cartodb.CDB_CONF ( CONF_NAME TEXT, CONF_KEY TEXT, CONF_VALUE TEXT NOT NULL, PRIMARY KEY (CONF_NAME, CONF_KEY));
```